pkg/kv/kvserver/batcheval/cmd_export_test.go

```diff
@@ -350,20 +364,21 @@ INTO
 		expect(t, res7, 1, 1, 1, 1)
 		latestRespHeader = kvpb.ResponseHeader{
 			ResumeSpan: &roachpb.Span{
-				Key:    []byte(fmt.Sprintf("/Table/%d/1/2", tableID)),
-				EndKey: []byte("/Max"),
+				Key:    sKey(fmt.Sprintf("/Table/%d/1/2", tableID)),
+				EndKey: sKey("/Max"),
```

Comment:
Wait how is that even supposed to work?

`sKey()` takes a string as argument _but does not interpret it_.
So this is literally encoding the bytes `'/', 'T', 'a', 'b'...` as start key, and the bytes `'/', 'M', 'a'..` as end key.

'M' sorts before 'T', so this span is really empty.

Did you mean to construct proper table prefixes and keys.KeyMax here?

Reply:
> Did you mean to construct proper table prefixes and keys.KeyMax here?

I did not. 

If look at the code _before_ my change, it is using these literal strings `/Max` and `/Table`, passing them as the "expected" arguments to comparisons. My changes here are not changing that, just adding `/Tenant/x` as needed.

I started down the path of using `keys.Rewrite.*ToTenantPrefix`, and which then would have meant switching the literals to e.g. `keys.KeyMax` , but this was a more invasive change to the existing test than is required just to remove the tenant/non-tenant limit.